jbmc/regression/jbmc/NondetEnumArgField/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+NondetEnumArgField.class
+--function NondetEnumArgField.test --cp `../../../../scripts/format_classpath.sh . ../../../src/java_bytecode/library/core-models.jar`
```

Comment:
why can't the field be `null`?

Reply:
It can, I just wanted it to assert that it can in fact come up with a non-null enum.